modules/release-notes/ref-release-notes-new-features.adoc

```diff
@@ -27,26 +27,32 @@ This update removes the `kube-rbac-proxy` sidecar container from the RHDH Operat
 [id="feature-rhidp-4414"]
 == Identifying Backstage flavor for plugins by using the `developerHub.flavor` field
 
-With this update, you can use the `developerHub.flavor` field to identify whether plugins are running on {product-very-short}, {rhtap-very-short}, or vanilla Backstage, as shown in the following example:
-
-.`app-config.yaml` fragment with the `developerhub.flavor` field
-
-[source,yaml,subs="+quotes"]
-----
-developerHub:
-  flavor: <flavor>
-----
-
-`flavor`::
+With this update, you can use the `developerHub.flavor` field to identify whether plugins are running on {product-very-short}, {rhtap-very-short}, or vanilla Backstage, as shown in the following example:
+
+.`app-config.yaml` fragment with the `developerhub.flavor` field
+
+[source,yaml,subs=&#34;+quotes&#34;]
+----
+developerHub:
+  flavor: &lt;flavor&gt;
+----
+
+`flavor`::
 Identify the flavor of Backstage that is running. Default value: `rhdh`
 
 [id="feature-rhidp-4419"]
 == Ability to manage Persistent Volume Claim (PVCs) in {product-very-short} Operator
 
-You can now mount directories from pre-created PersistentVolumeClaims (PVCs) using the `spec.application.extraFiles.pvcs` field, while configuring RHDH Operator.
-For more information, see link:https://docs.redhat.com/en/documentation/red_hat_developer_hub/{product-version}/html-single/configuring/index#configuring-the-deployment[Configuring {product} deployment when using the Operator].
+You can now mount directories from pre-created PersistentVolumeClaims (PVCs) using the `spec.application.extraFiles.pvcs` field, while configuring RHDH Operator.
+For more information, see link:https://docs.redhat.com/en/documentation/red_hat_developer_hub/{product-version}/html-single/configuring/index#configuring-the-deployment[Configuring {product} deployment when using the Operator].
 
 
+[id="feature-rhidp-4780"]
+== [Docs] Document per-ConfigMap/Secret configuration of mountPath 
```

Comment:
this item is new after regenerating. See https://issues.redhat.com/browse/RHIDP-4780 for the weirdly formatted release note text field. cc: @pabel-rh 

Perhaps the `Release Note Status` should be `Rejected` ?

As we already have these: 
* https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.4/html/release_notes/new-features#enhancement-rhidp-2200
* https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.4/html-single/configuring/index#mounting-additional-files-in-your-custom-configuration-using-rhdh-operator